tests/test_sailor/test_assetcentral/test_equipment.py

```diff
@@ -98,6 +98,33 @@ def test_location_different_instances_always_fetch(self, mock_find):
 
         mock_find.assert_has_calls(expected_calls)
 
+    @patch('sailor.assetcentral.equipment._create_or_update_notification')
+    def test_create_notification_builds_request(self, create_update_mock):
+        equipment = Equipment({'equipmentId': '123', 'location': 'Walldorf'})
```

Comment:
I am not keen on changing it here, because the whole test_equipment module does not use it. and it does not really help anything as the tests really do with very little input for the equipment objects.